include/Zydis/DecoderTypes.h

```diff
@@ -1253,9 +1253,11 @@ typedef struct ZydisDecodedInstruction_
 typedef struct ZydisDecoderContext_
 {
     /**
-     * A pointer to the internal instruction definition.
+     * The instruction definition id.
+     *
+     * Only valid in combination with the instruction encoding.
      */
-    const void* definition;
+    ZyanU16 definition_id;
```

Comment:
This change allows us to actually fuzz the code (open todo) and as well guarantees we never read invalid memory, even if the user passes nonsense data.